MdePkg/Library/BasePanicLibNull/PanicLib.c

```diff
@@ -0,0 +1,33 @@
+/** @file
+  Base Panic Library with empty functions.
+
+  Copyright (c) Microsoft Corporation.
+

```

Comment:
Need SPDX identifier.

```suggestion
    SPDX-License-Identifier: BSD-2-Clause-Patent
```

Reply:
Fixed.